app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,6 @@
     <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
 
     <%= stylesheet_link_tag("application") %>
-    <%= stylesheet_link_tag("tailwind", "data-turbo-track": "reload") %>
```

Comment:
shouldn't this now link the deprecated generated file?

Reply:
It's compiled into the application.css now.

Reply:
I did check the views that rely on `tw-` classes and they look the same as before, e.g. trusted publishing.